www/includes/easyparliament/templates/html/donate/_stripe_donate.php

```diff
@@ -1,53 +1,20 @@
 <?php
 
-# define array of payment amounts and current/one-off options
-# note - new annual and monthly payments need to be defined as new prices in stripe
-# e.g. 'donate_monthly_10'.
-$payment_amounts = [
-    'monthly' => [
-        '2' => '£2',
-        '5' => '£5',
-        '10' => '£10',
-    ],
-    'annually' => [
-        '10' => '£10',
-        '50' => '£50',
-        '100' => '£100',
-    ],
-    'one-off' => [
-        '5' => '£5',
-        '10' => '£10',
-        '20' => '£20',
-    ],
-];
-
-$default_amounts = [
-    'monthly' => '5',
-    'annually' => '10',
-    'one-off' => '10',
-];
-
-$default_type = 'one-off';
-
-# use the how-often parameter if set, if not default to option at end of line (options are 'monthly', 'annually', or 'one-off')
-$initial_payment_type = get_http_var('how-often', $default_type);
-
-# use the how-much parameter if set, if not default to default amount for initial payment type
-$how_much = get_http_var('how-much', $default_amounts[$initial_payment_type]);
-
-# if how-much is not in the allowed values for the current payment type, set to 'other', and set $other_how_much to the value of how-much
-if (!array_key_exists($how_much, $payment_amounts[$initial_payment_type] ?? [])) {
-    $how_much = 'other';
-    $other_how_much = get_http_var('how-much');
-} else {
-    $other_how_much = '';
-}
+
 
 ?>
 <script src="https://www.google.com/recaptcha/api.js" async defer></script>
 
 <a name="donate-form"></a>
-<form class="donate-form" method="post" name="donation_form">
+
+<button type="button" id="default_donate_button" class="button button-primary button--large" style="margin-right: 10px" onclick="restrict_to_default('<?= $how_much ?>', '<?= $payment_type ?>')">
+        Donate <?= $verbose_amount ?>
+</button>
+
+<input type="button" value="Donate another amount" class="open-form-button button button-primary button--large" >
+    
+
+<form class="donate-form" method="post" name="donation_form" style="display: None";>
```

Comment:
Please have the default be visible, and use `html.js` in a stylesheet parent to set it to display none if JS is available. Otherwise there's no way to see the form if the JS doesn't work.

Reply:
There's a general thing here that the stripe form needs javascript - but can't remember if I moved the warning about this *out* of the now invisible form. Either way, yes , can fix.